src/lib/ggpo/network/udp.cpp

```diff
@@ -15,10 +15,13 @@ CreateSocket(int bind_port, int retries)
    sockaddr_in sin;
    int port;
    int optval = 1;
+   struct linger loptval;
+   loptval.l_onoff = 0;
+   loptval.l_linger = 0;
 
    s = socket(AF_INET, SOCK_DGRAM, 0);
    setsockopt(s, SOL_SOCKET, SO_REUSEADDR, (const char *)&optval, sizeof optval);
-   setsockopt(s, SOL_SOCKET, SO_DONTLINGER, (const char *)&optval, sizeof optval);
```

Comment:
I have a question about the intended and actual behavior of this line. The docs at https://docs.microsoft.com/en-us/windows/win32/api/winsock/ns-winsock-linger say that calling setsockopt with `SO_DONTLINGER` and optval 1 is equivalent to calling setsockopt with `SO_LINGER` and optval `struct linger{1, 1}`. This seems obviously wrong though, like turning ON the DONTLINGER setting should turn OFF the LINGER setting.....

So, you do want to turn off LINGER, right?

Reply:
Well, the docs at https://docs.microsoft.com/en-us/windows/win32/api/winsock/nf-winsock-setsockopt say the opposite, so I'll just assume it's a problem with the docs....